downstream/modules/automation-hub/proc-delete-collection.adoc

```diff
@@ -0,0 +1,22 @@
+
+[id="delete-collection"]
+
+= Deleting a collection on {HubName}
+
+You can further manage your collections by deleting unwanted  collections, provided that the collection is not dependant on other collections. Click the *Dependencies* tab on a collection to see a list of other collections that uses the current collection.
```

Comment:
I think this section requires a NOTE about collection dependencies. Even tho its in the UI its probably worth bringing some attention to that there's a degree of risk inherent in this procedure.

Reply:
Would the risks be that users might delete something with dependencies and potentially break stuff? because I think the UI prevents users from deleting stuff that has dependencies

Reply:
Ok yeah, so long as the UI throws the warning we're good.